libraries/rush-sdk/webpack.config.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-env es6 */
+'use strict';
+
+const webpack = require('webpack');
+const { PackageJsonLookup } = require('@rushstack/node-core-library');
+const { PreserveDynamicRequireWebpackPlugin } = require('@rushstack/webpack-preserve-dynamic-require-plugin');
+const { DeepImportsPlugin } = require('@rushstack/webpack-deep-imports-plugin');
+
+module.exports = () => {
+  const packageJson = PackageJsonLookup.loadOwnPackageJson(__dirname);
+
+  const externalDependencyNames = new Set([
+    ...Object.keys(packageJson.dependencies || {})
+    // ...Object.keys(packageJson.peerDependencies || {}),
+    // ...Object.keys(packageJson.optionalDependencies || {}),
+    // ...Object.keys(packageJson.devDependencies || {})
+  ]);
+
+  return {
+    context: __dirname,
+    mode: 'development', // So the output isn't minified
+    devtool: 'source-map',
+    entry: {
+      'rush-sdk': `${__dirname}/lib-esnext/index.js`
+    },
+    output: {
+      path: `${__dirname}/dist`,
+      filename: '[name].js',
+      chunkFilename: 'chunks/[name].js',
+      library: {
+        type: 'commonjs2'
+      }
+    },
+    target: 'node',
+    plugins: [
+      new PreserveDynamicRequireWebpackPlugin(),
+      new webpack.ids.DeterministicModuleIdsPlugin({
+        maxLength: 6
+      }),
+      new DeepImportsPlugin({
+        path: `${__dirname}/temp/rush-sdk-manifest.json`,
+        inFolderName: 'lib-esnext',
+        outFolderName: 'lib',
+        pathsToIgnore: [],
+        dTsFilesInputFolderName: 'lib-commonjs'
+      })
+    ],
+    externals: [
+      ({ request }, callback) => {
```

Comment:
This rule needs to expressly exclude `@microsoft/rush-lib` or else it will be trying to load `rush-lib` in order to figure out how to load `rush-lib`.